app/crud.py

```diff
@@ -27,10 +30,28 @@ def create_user(db: Session, user: UserBase):
     return db_user
 
 
+def update_user_by_token(db: Session, token: str):
```

Comment:
a more descriptive function name such as `update_user_last_used_field` would be better in my opinion

Reply:
ok :heavy_check_mark: